src/autocomplete/autocomplete.ts

```diff
@@ -116,13 +116,17 @@ export class Autocomplete {
       this.service.suggest(value)
         .then(suggestions => {
           this.index = -1;
-          this.suggestions.splice(0, this.suggestions.length, ...suggestions);
-          if (suggestions.length === 1 && this.type !== autocompleteType.Suggest) {
-            this.select(suggestions[0]);
-          } else if (suggestions.length === 0) {
+          if (!suggestions) {
             this.collapse();
           } else {
-            this.expanded = true;
+            this.suggestions.splice(0, this.suggestions.length, ...suggestions);
+            if (suggestions && suggestions.length === 1 && this.type !== autocompleteType.Suggest) {
```

Comment:
maak nie veel uit, maar moet die check extra check op suggestions er wel bij als we hier in de 'else ' zitten van 'if (!suggestions)...'?